Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessSimpleMetadata.java

```diff
@@ -86,6 +87,12 @@ public boolean isUndefined() {
     }
 
     public boolean isRequired() {
+        if (container != null && container.getChildMetadata().isEmpty()) {
```

Comment:
The required simple metadata of a 'docType' are also no longer required.
The simple metadata also has a container wich is the selected docType (volume...) , so this condition "Container != null && container.getChildMetadata().isEmpty()" is not precise enough to distinguish between simple metadata of  the selected docType or simple metadata of a metadata group.

Reply:
Since a docType is “simple”, it shouldn't have a container, or am I misunderstanding something?

Can you give me an example (a ruleset) where the problem occurs?

Reply:
The doctype itself doesn't have a container, but the simple metadata like "HaupTitel" has a container, which is the doctype (e.g. volume)  and where all the metadata of the doctype are located.